pkg/relayer/relay_authenticator/relay_verifier.go

```diff
@@ -91,6 +91,39 @@ func (ra *relayAuthenticator) VerifyRelayRequest(
 	return ErrRelayAuthenticatorInvalidSessionSupplier
 }
 
+// CheckRelayRewardEligibility verifies the relay's session hasn't expired for reward
```

Comment:
#PUC so the community can understand what's happening with the windows here too.